components/input/Password.tsx

```diff
@@ -84,8 +84,9 @@ const Password = React.forwardRef<InputRef, PasswordProps>((props, ref) => {
         // https://github.com/ant-design/ant-design/issues/23524
         e.preventDefault();
       },
+      disabled,
     };
-    return React.cloneElement(React.isValidElement(icon) ? icon : <span>{icon}</span>, iconProps);
+    return React.cloneElement(<span>{icon}</span>, iconProps);
```

Comment:
Hi! Are you sure that React.isValid checking is no longer needed?

Reply:
but how can i fix this problem 
if you don't warp a layer on the icon 
you can't add disabled attribute on icon
many input element have this problem
like this : https://codesandbox.io/p/sandbox/qian-zhui-he-hou-zhui-antd-5-12-4-forked-g5d95n